extensions/smartthings-connector/.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
Are you using an old version? New plugins should have `.eslintrc.json` with this

```
{
  "root": true,
  "extends": ["@raycast"]
}
```

Reply:
> Are you using an old version? New plugins should have `.eslintrc.json` with this
> 
> ```
> {
>   "root": true,
>   "extends": ["@raycast"]
> }
> ```

Hi,
This was the best running version for my use case. Is it necessary to change the lint config?

Reply:
What was not working in a newer version of Raycast?

Reply:
Changed the version and worked now just fine :)
<img width="452" alt="image" src="https://github.com/user-attachments/assets/a80ed805-f30d-488b-aea6-18a5c7f4b276">